src/app/pages/PerpetualPage/hooks/usePerpetual_accountBalance.ts

```diff
@@ -12,24 +21,46 @@ type AccountBalance = {
 
 export const usePerpetual_accountBalance = (pairType: PerpetualPairType) => {
   const blockId = useBlockSync();
+  const account = useAccount();
   const [data, setData] = useState<AccountBalance>({
     total: '',
     available: '',
     inPositions: '',
     unrealized: '',
   });
 
-  // TODO: implement perpetual account Data fetching
+  const [availableBalance, setAvailableBalance] = useState('0');
+
+  const traderState = usePerpetual_queryTraderState();
+  const ammState = usePerpetual_queryAmmState();
+  const marginBalance = usePerpetual_marginAccountBalance();
+
+  useEffect(() => {
+    bridgeNetwork
+      .call(
+        Chain.BSC,
+        getContract('PERPETUALS_token').address,
+        marginTokenAbi,
+        'balanceOf',
+        [account],
+      )
+      .catch(e => console.log(e))
+      .then(result => result && setAvailableBalance(String(result)));
```

Comment:
I don't know why but I get vastly different values from this implementation and calling it on bscscan.
https://testnet.bscscan.com/token/0xDaF779963E49c081dB6cf3Cf60368B0075517010

Could it be an RPC problem again?

Reply:
I am not sure but I see the correct number:
![image](https://user-images.githubusercontent.com/14878713/142842944-391f697b-10c3-4e43-9e80-e31b74ab3c9a.png)

I would say that we should merge it as it is and we'll fix it in the future if it's a problem (because I currently don't see anything odd on local).

Reply:
Yeah It probably is "just" a RPC out of sync problem...